mw-webapp/src/component/header/Header.module.scss

```diff
@@ -1,6 +1,11 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$gapHeaderExtraLarge: 100px;
```

Comment:
Is it unique gap? Please check/ If yes then lets discuss with designers do we really need it) f not then need create global var

Reply:
waiting for a response from the designer

Reply:
after response add new global variable $gapExtraExtraLarge: 96px;